locations/spiders/winterthur_ch.py

```diff
@@ -42,6 +42,10 @@ def parse_playgrounds(self, response):
             item = {
                 "lat": lat,
                 "lon": lon,
+                "street": props.get("streetName"),
+                "housenumber": props.get("houseNo"),
+                "city": "Winterthur",
+                "country": "CH",
```

Comment:
There is middleware that will populate `item["country"]` automatically from the `_ch` in the spider name. Thus no need to specify the country explicitly. Not a big deal, just a style preference that seems to have come into existence.

Reply:
I tend to prefer it sitting explicitly in the spider, personally, so this is fine 👍🏻

Reply:
I prefer it too :)